src/api/graphics.c

```diff
@@ -339,6 +339,15 @@ static int l_lovrGraphicsGetDimensions(lua_State* L) {
   return 2;
 }
 
+static int l_lovrGraphicsGetHasWindow(lua_State *L) {
+  bool window = false;
+#ifndef NO_WINDOW
```

Comment:
I ended up folding the NO_WINDOW define into the platform layer, so in this case I think `platform/android.c` should implement `lovrPlatformHasWindow` and always return `NULL`, instead of using an ifdef here.